### ISSUE TITLE
[Feature Request] Dont Plot Into Same Folder/Drive as Phase 1

### ISSUE THREAD

**Issue by Wontell** [maded2/plotng#51]
Hello,
Can this feature be considered please?
Example 7 Parallel plots on 7 drives staggered 3 times, with Maximum Phase1 plots set to 7

When one of the first 7 plots finished a new staggered one should start on the drive that Finished Phase 1.
Right now it will start on the next drive on the list, which is the first one since list restarted.

Regards

**Comment by maded2** [comment]
you can run 2 instances of PlotNG to do this, just use different port number

**Comment by Wontell** [comment]
Hi,
The second instance doesn't knows when Phase 1 ends on the first instance.

Its like this

DDDDDDD [7 drives and 7 plots running Phase 1]
when Phase2 ends on ANY drive start new Plot
so if
DDDODDD [the O is the one that finished Phase one, it should start on that one and then on the next one that finished Phase one

This is the ultimate stagger, you can have as many plots running parallel, and when any one finishes first Phase1 a new plot starts there.

Basically you need to add an option that says if Phase1 runs on drive, do not start plotting there
Your app already shows phases

**Comment by maded2** [comment]
Try this configuration:

MaxActivePlotPerTemp = 3
MaxActivePlotPerPhase1 = 7
NumberOfParallelPlots = 21

**Comment by Wontell** [comment]
> Try this configuration:
> 
> MaxActivePlotPerTemp = 3
> MaxActivePlotPerPhase1 = 7
> NumberOfParallelPlots = 21

Hi
 it wont work

When Phase 1 is finished on one drive, the plotNG just moves to next TEMP folder on the list based on its settings
So if Drive 5 finished Phase1, but its Drive 1 turn for the Temp, it will start plotting on Drive 1.

Because not all drives are same speed some plots can start after others but finish first, so doing it manually wont work, you never know which out of the 7 running will finish Phase1 first.

if you add such option "MaxActivePhase1PerTemp" it will solve this issue and the Plotter will start plotting  on a drive that finished Phase 1 and not the next one the list of temp folders.

**Comment by maded2** [comment]
Sorry, it really sound like you have a specific setup that needed specific custom coding for.   I welcome any PR from you, but I don't think I have the time to implement.
